schedules/delete_schedule.py

```diff
@@ -0,0 +1,19 @@
+import asyncio
+
+from temporalio.client import Client
+
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle = client.get_schedule_handle(
+        "workflow-schedule-id",
+    )
+    desc_handle = await handle.describe()
+    print(f"{desc_handle.schedule}")
+
+    await handle.delete()
+    print(f"State: {desc_handle.schedule.state}")
```

Comment:
A handles state is not automatically updated after delete, you would have to refetch it. But then you would get a not-found. Regardless, doesn't have value to print the pre-deleted state of a schedule after you've deleted it